src/features/MentorPage/components/MentorList/MentorCard/Expanded/Header.tsx

```diff
@@ -36,33 +60,44 @@ export const Header = ({ mentor, onDismiss }: Props) => {
       </HeaderWrapper>
     </Container>
   ) : (
-    <Container isAvailable={!mentor.isVacationing} isMobile={false}>
-      <Availability isShowing={availabilityMessage.length > 0}>
-        {availabilityMessage}
-      </Availability>
-      <ProfilePicture isMobile={false} />
-      <BasicInfo mentor={mentor} />
+    <Container isAvailable={!mentor.isVacationing} isMe={isMe} isMobile={false}>
+      <Tag status={getStatus(isMe, isAvailable, isNew)} />
+      <ProfilePicture isMe={isMe} isMobile={false} />
+      <BasicInfo isMe={isMe} mentor={mentor} />
     </Container>
   );
 };
 
-const Container = styled.div<{ isAvailable: boolean; isMobile: boolean }>`
-  background-color: ${({ isAvailable }) =>
-    isAvailable ? palette.purple : palette.blueGrey};
+const Container = styled.div<{
+  isAvailable: boolean;
+  isMe: boolean;
+  isMobile: boolean;
+}>`
+  background-color: ${({ isAvailable, isMe }) => {
+    if (!isAvailable) {
+      return palette.blueGrey;
+    }
+    if (isMe) {
+      return palette.blue;
+    } else {
+      return palette.purple;
+    }
+  }};
```

Comment:
I would like to suggest a small refactor to make the code a bit easier to read.

1. Instead of if-else and multiple boolean-flags, we will give the Container-element just the color.
2. The color will be detemined based on the status we get from getStatus-function using a new map

```javascript
  const status = getStatus(isMe, isAvailable, isNew)

  const statusColors = {
    'me': palette.blue,
    'unavailable': palette.blueGrey,
    'new': palette.purple,
    'empty': palette.purple
  }

  return isMobile ? (
    <Container statusColor={statusColors[status]} isMobile>
```

Then the Container-element code gets quite a bit simpler:

```javascript
 const Container = styled.div<{
  statusColor: string
  isMobile: boolean
}>`
  background-color: ${({ statusColor }) => statusColor
  };
```

Reply:
Thank you, this made it more simple!